packages/ERTP/src/paymentLedger.js

```diff
@@ -62,19 +62,19 @@ export const makePaymentLedger = (
 
   /**
    * Methods like deposit() have an optional second parameter
-   * `optAmountPattern`
+   * `optAmountShape`
```

Comment:
Is there some discussion somewhere of the terminological shift from `xxxPattern` to `xxxShape`?

Reply:
Will shortly be in store/src/types.js once #4214 is merged. Resolving.